src/implementation.cpp

```diff
@@ -47,16 +48,22 @@ std::string href_from_file(std::string_view input) {
 }
 
 bool can_parse(std::string_view input, const std::string_view* base_input) {
-  ada::result<ada::url_aggregator> base;
+  ada::url_aggregator base_aggregator;
   ada::url_aggregator* base_pointer = nullptr;
+
   if (base_input != nullptr) {
-    base = ada::parse<url_aggregator>(*base_input);
-    if (!base) {
+    base_aggregator = ada::parser::parse_url_impl<ada::url_aggregator, false>(
```

Comment:
Let's not forget that, since we don't have `parse_url_impl<ada::url, false>` we won't get this function signature in any of the static releases.

Are we ok with only exposing this with `ada::url_aggregator`? @lemire

Reply:
I am not sure I understand the issue, but we should be very deliberate before breaking anything.

You do not want to get angry developers asking you why you pulled backward compatibility. 🥊